.github/workflows/publish.yml

```diff
@@ -26,8 +23,11 @@ jobs:
       - name: Build
         run: yarn build
 
+      - name: Reset git status
+        run: git reset --hard # workaround for types changes after building
```

Comment:
@wooorm @ChristianMurphy I add this as workaround temporarily before we determine what's the problem with types after building, how do you think?

Reply:
Maybe add a link in the comment to the action job that had these LF errors?

Otherwise looks good

Reply:
> Maybe add a link in the comment to the action job that had these LF errors?

The action job log will be deleted after days. Or add `git status` command before resetting so that we can always understand what is happening.

Reply:
ah, true, never mind!